integration/containerd/runk/runk-tests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2021-2022 Kata Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will validate runk with containerd
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+dir_path=$(dirname "$0")
+source /etc/os-release || source /usr/lib/os-release
+source "${dir_path}/../../../lib/common.bash"
+source "${dir_path}/../../../.ci/lib.sh"
+
+KATACONTAINERS_REPO=${katacontainers_repo:="github.com/kata-containers/kata-containers"}
+RUNK_SRC_PATH="${GOPATH}/src/${KATACONTAINERS_REPO}/src/tools/runk"
+RUNK_BIN_PATH="/usr/local/bin/runk"
+TEST_IMAGE="docker.io/library/busybox:latest"
+CONTAINER_ID="id1"
+PID_FILE="${CONTAINER_ID}.pid"
+
+setup() {
+    # can't find cargo when make runk below, so we install rust to make runk build correctly
+    "${dir_path}/../../../.ci/install_rust.sh" && source "$HOME/.cargo/env"
+    echo "restart containerd service"
+    sudo systemctl restart containerd
+    echo "pull container image"
+    sudo ctr images pull ${TEST_IMAGE}
```

Comment:
you should also use https://github.com/kata-containers/tests/blob/b77a49f6bcc4fab38a513fcf7cf9ee1e13c59d40/metrics/lib/common.bash#L67

Reply:
Ok, I will use it .  But why is it under matrics directory? Maybe moving to global lib is better.